tastypie/__init__.py

```diff
@@ -3,9 +3,10 @@
 
 __author__ = 'Daniel Lindsley & the Tastypie core team'
 
-# This variable should include the Django-tastypie version number (indexes 0,1,2) and the Grok version number for our fork (from index 3).
+# This variable should include the Django-tastypie version number (indexes 0,1,2) and the Grok version for our fork (from index 3).
 # This is required so that we have a distinct version number for each internal Grok version of Django-tastypie, so pip knows to update tastypie when running the app repo.
-VERSION = (0, 14, 2, '-grokfork-0.1')
+# We need the format to ensure that __version__ forms a local version identifier: https://www.python.org/dev/peps/pep-0440/#local-version-identifiers
+VERSION = (0, 14, 2, '+grokfork.0.1')
```

Comment:
Unfortunately, I believe this will generate 0.14.2.+grokfork-0.1, which doesn't obey the PEP. You should be able to use pip install to see what it believe on the directory.